assets/bigquery/fishing-events.sql.j2

```diff
@@ -352,7 +351,7 @@ WITH
   # Add spatial measures region information
   #
   {% with complete_events_table="complete_fishing_event", regions_table=regions_table %}
-    {% include 'regions-subquery.sql.j2' %}
+    {% include 'regions-subquery-fishing.sql.j2' %}
```

Comment:
Is the `regions-subquery.sql.j2` still in use?, if not I'd remove it.

Reply:
@rdgfuentes yes we are using it in the other events (encounters, port-visits, loitering)